llvm/docs/LangRef.rst

```diff
@@ -3394,17 +3394,41 @@ Floating-Point Environment
 The default LLVM floating-point environment assumes that traps are disabled and
 status flags are not observable. Therefore, floating-point math operations do
 not have side effects and may be speculated freely. Results assume the
-round-to-nearest rounding mode.
+round-to-nearest rounding mode, and subnormals are assumed to be preserved.
```

Comment:
The `denormal-fp-math` attribute does affect whether subnormals are preserved, although I suspect it is somewhat inconsistently applied at the moment.

Reply:
I made the text mention this attribute now, is that better?

Reply:
I think this works for now.